packages/prettier-plugin-liquid/src/printer/printer-liquid-html.ts

```diff
@@ -565,6 +565,10 @@ function printNode(
       return printLiquidDocExample(path as AstPath<LiquidDocExampleNode>, options, print, args);
     }
 
+    case NodeTypes.LiquidDocDescriptionNode: {
+      return '';
```

Comment:
This appeases the build - we will implement this when we build out prettier support